package.json

```diff
@@ -17,8 +17,11 @@
   },
   "license": "MIT",
   "main": "index.js",
+  "files": [
+    "index.js"
+  ],
   "scripts": {
-    "lint": "eslint .",
+    "lint": "eslint . && stylelint --ignore-path .gitignore \"**/*.css\"",
```

Comment:
Perhaps also specify the ignore path of eslint and remove `.eslintignore` from the root?

Reply:
Also, remove the old `docs/` folder from the `.gitignore`, it's a leftover.